alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -2967,6 +2967,16 @@ public static function addDeliveryTagToEntry(entry $entry)
 			$entry->save();
 		}
 	}
+
+	public static function handleDelayedNotification(entry $entry)
+	{
+		$delayedLockedJob = BatchJobLockPeer::retrieveByEntryIdAndStatus($entry->getEntryId(), BatchJob::BATCHJOB_STATUS_DELAYED);
+		if ($delayedLockedJob)
```

Comment:
check in the template what is the condition. cant release the job when entry is ready, while the conditioon is something else